Source/Core/VideoCommon/RenderState.h

```diff
@@ -9,10 +9,54 @@
 #include "VideoCommon/BPMemory.h"
 #include "VideoCommon/BPStructs.h"
 
+enum class PrimitiveType : u32
+{
+  Points,
+  Lines,
+  Triangles,
+  TriangleStrip,
+};
+
+union RasterizationState
+{
+  void Generate(const BPMemory& bp, PrimitiveType primitive_type);
+
+  RasterizationState& operator=(const RasterizationState& rhs);
```

Comment:
I wonder why we need this operator explicitly defined here. Isn't it the same as the implicit operator= ?